packages/launch-config/test/debug-config/workspaceManager.test.ts

```diff
@@ -85,70 +104,39 @@ describe('launchConfig Unit Tests', () => {
 
         it('should create a launch config for non-workspace apps', () => {
             const mockProjectPath = '/mock/project/path';
-            const mockProjectName = 'project';
-            const mockTargetFolder = '/target/folder';
             (isFolderInWorkspace as jest.Mock).mockReturnValue(false);
             (handleAppsNotInWorkspace as jest.Mock).mockReturnValue({
                 launchJsonPath: mockProjectPath,
                 cwd: '${workspaceFolder}'
             });
 
-            const result = handleSavedWorkspace(
-                mockProjectPath,
-                mockProjectName,
-                mockTargetFolder,
+            const options = {
                 isAppStudio,
-                mockVscode
-            );
+                vscode: mockVscode
+            } as DebugOptions;
+            const result = handleWorkspaceConfig(mockProjectPath, options);
             expect(result).toEqual({
                 launchJsonPath: mockProjectPath,
                 cwd: '${workspaceFolder}'
             });
         });
     });
 
-    describe('handleOpenFolderButNoWorkspaceFile', () => {
-        it('should create a launch config for non-workspace apps if folder is not in workspace', () => {
-            const mockProjectPath = '/mock/project/path';
-            const mockTargetFolder = '/target/folder';
-            (isFolderInWorkspace as jest.Mock).mockReturnValue(false);
-            (handleAppsNotInWorkspace as jest.Mock).mockReturnValue({
-                launchJsonPath: mockProjectPath,
-                cwd: '${workspaceFolder}'
-            });
-
-            const result = handleOpenFolderButNoWorkspaceFile(
-                mockProjectPath,
-                mockTargetFolder,
-                isAppStudio,
-                mockVscode
-            );
-            expect(result).toEqual({
-                launchJsonPath: mockProjectPath,
-                cwd: '${workspaceFolder}'
-            });
-        });
-
-        it('should update paths for nested folders inside an open folder', () => {
+    describe('handleUnsavedWorkspace', () => {
+        it('should update paths for nested folders inside a workspace', () => {
```

Comment:
These tests are failing locally for me on Windows because of the forward slashes. Could you please use join() for string paths ?